cfme/containers/image.py

```diff
@@ -133,23 +163,24 @@ def step(self):
     def resetter(self):
         # Reset view and selection
         self.view.toolbar.view_selector.select("List View")
-        self.view.paginator.reset_selection()
+        self.view.paginator.set_items_per_page(1000)
```

Comment:
**Required/Question**

Is there a reason for increasing this limit every time the page is visited?

Reply:
We have problem with very slow navigation to Container Image Details ,
According to Pete and Ievgen the problem related to uses a lot of selenium calls because of older Table usage in 5.8 etc .
One of Ievgen suggestion , for try to improve it little beat by set item per page to 1000 , in order to save the moving from page to page in order to get to the right image.

Reply:
There is a 'use_search' boolean for `entities.get_entity` now, but at the moment it only supports name search, which doesn't look like it will work for details.

I think you should move this setting into the Details navigation step, above L169, so that the limit is only changed when actually navigating to 'Details', and 'All' navigation isn't slowed by this.